packages/fuchsia_remote_debug_protocol/lib/src/dart/dart_vm.dart

```diff
@@ -0,0 +1,180 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:json_rpc_2/json_rpc_2.dart' as json_rpc;
+import 'package:web_socket_channel/io.dart';
+
+import '../common/logging.dart';
+
+const Duration _kConnectTimeout = const Duration(seconds: 30);
+
+const Duration _kReconnectAttemptInterval = const Duration(seconds: 3);
+
+const Duration _kRpcTimeout = const Duration(seconds: 5);
+
+final Logger _log = new Logger('DartVm');
+
+/// Definition of an asynchronous function for connecting to a `Uri`.
+///
+/// Creates a JSON RPC-2 connection.
+typedef Future<json_rpc.Peer> RpcPeerConnectionFunction(Uri uri);
+
+/// `DartVm` uses this function to connect to the Dart VM on Fuchsia.
```

Comment:
use square brackets to refer to classes - `[DartVm]`.

Reply:
Done.